manifests/worker.pp

```diff
@@ -75,7 +87,9 @@
   $symlinks       = undef,
   $release        = 'mitaka',
   $gerrit_user    = undef,
-  $gerrit_email   = undef ) {
+  $gerrit_email   = undef, 
```

Comment:
This is a bit silly, but just to keep lint happy: there is a trailing whitespace here.